.gitignore

```diff
@@ -46,7 +46,7 @@ playground.xcworkspace
 # you should judge for yourself, the pros and cons are mentioned at:
 # https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
 #
-# Pods/
+Pods/
```

Comment:
I think I am personally okay with this change, but I wanted to understand more the reasoning behind it. Also would like to hear others' (at least @tjprescott) opinions

Reply:
I was torn on this one. On one hand committing the sources to the tree helps insulate us from unreliable dependencies that might get yanked and that might also get changed underneath us (if they are depending on a tag instead of a commit SHA).

However it is a lot of bloat to add into our repository and I'm not sure what the ramifications are from a licensing perspective either. The other thing that I want to avoid is the temptation for someone to go in and change anything in the ```Pods/``` folder since I think that could be a nightmare.

I don't think adding the ignore here is the end of the story. We really do need to consider the potential security ramifications for our engineering system.

Reply:
https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control

Reply:
Based on this, I am okay ignoring the Pods folder as well. We don't commit the dependencies in other SDKs.